src/main/java/edu/kit/ipd/crowdcontrol/objectservice/ExperimentController.java

```diff
@@ -0,0 +1,120 @@
+package edu.kit.ipd.crowdcontrol.objectservice;
+
+import edu.kit.ipd.crowdcontrol.objectservice.crowdworking.PlatformManager;
+import edu.kit.ipd.crowdcontrol.objectservice.crowdworking.TaskOperationException;
+import edu.kit.ipd.crowdcontrol.objectservice.event.ChangeEvent;
+import edu.kit.ipd.crowdcontrol.objectservice.event.Event;
+import edu.kit.ipd.crowdcontrol.objectservice.event.EventManager;
+import edu.kit.ipd.crowdcontrol.objectservice.event.EventObservable;
+import edu.kit.ipd.crowdcontrol.objectservice.proto.Experiment;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import rx.Observable;
+import rx.Observer;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Created by lucaskrauss at 05.02.2016
+ */
+public class ExperimentController implements Observer<Event<Experiment>> {
+
+    private final Logger log = LogManager.getLogger(ExperimentController.class);
+    private final Observable<Event<Experiment>> observable = EventManager.EXPERIMENT_CREATE.getObservable();
```

Comment:
The current state is that the conroller get informed if the `put`-request was performed, which emits the EXPERIMENT_CREATE event. The controller itself doesn't emit any events on startup